_maps/galactica.json

```diff
@@ -0,0 +1,17 @@
+{
+	"map_name": "NSV Galactica",
+	"map_link": "Galactica",
```

Comment:
```suggestion
	"map_link": null,
```
Keep this null until the map is properly added to the webmap registrar, so that it properly reports that no webmap exists,

Reply:
Changes implemented.